pkg/sloop/storemanager/storemanager.go

```diff
@@ -89,15 +94,17 @@ func (sm *StoreManager) gcLoop() {
 		metricGcRunCount.Inc()
 		before := time.Now()
 		metricGcRunning.Set(1)
-		_, err := doCleanup(sm.tables, sm.config.TimeLimit, sm.config.SizeLimitBytes, sm.stats)
+		cleanUpPerformed, numOfDeletedKeys, err := doCleanup(sm.tables, sm.config.TimeLimit, sm.config.SizeLimitBytes, sm.stats)
+		metricGcCleanUpPerformed.Set(boolToFloat(cleanUpPerformed))
+		metricGcDeletedNumberOfKeys.Set(numOfDeletedKeys)
 		metricGcRunning.Set(0)
 		if err == nil {
 			metricGcSuccessCount.Inc()
 		} else {
 			metricGcFailedCount.Inc()
 		}
 		metricGcLatency.Set(time.Since(before).Seconds())
-		glog.Infof("GC finished in %v with return %q.  Next run in %v", time.Since(before), err, sm.config.Freq)
+		glog.Infof("GC finished in %v with error %v.  Next run in %v", time.Since(before), err, sm.config.Freq)
```

Comment:
consider adding quotes around %v for the error - if the error string resolves to en empty string the result would look like: ` with error .  Next run `, implying an error occurred. The quotes would indicate the error message is blank.